src/NumericalAlgorithms/LinearOperators/Python/PowerMonitors.cpp

```diff
@@ -36,13 +36,18 @@ void bind_power_monitors(py::module& m) {
   bind_power_monitors_impl<1>(m);
   bind_power_monitors_impl<2>(m);
   bind_power_monitors_impl<3>(m);
-  m.def("convergence_rate",
-        py::overload_cast<const DataVector&, const size_t>(&convergence_rate),
-        py::arg("power_monitor"), py::arg("number_of_filtered_modes") = 0);
-  m.def("number_of_pile_up_modes",
-        py::overload_cast<const DataVector&, const size_t>(
-            &number_of_pile_up_modes),
-        py::arg("power_monitor"), py::arg("number_of_filtered_modes") = 0);
+  m.def(
+      "convergence_rate_and_number_of_pile_up_modes",
+      [](const DataVector& power_monitor,
+         const size_t number_of_filtered_modes) {
+        py::dict result{};
+        ConvergenceInfo info = convergence_rate_and_number_of_pile_up_modes(
```

Comment:
make const as clang-tidy suggests?

Reply:
Yep pretty sure I caught that one late (missed it when going through to make the fix), I will make sure clang tidy passes.